packages/opentelemetry-node/lib/elastic-node-sdk.js

```diff
@@ -58,20 +53,11 @@ class ElasticNodeSDK extends NodeSDK {
         //   `spanProcessor` nor `traceExporter` are passed in.
         /** @type {Partial<NodeSDKConfiguration>} */
         const defaultConfig = {
-            resourceDetectors: [
-                // Elastic's own detector to add some metadata
-                distroDetectorSync,
-                envDetectorSync,
-                processDetectorSync,
-                // hostDetectorSync is not currently in the OTel default, but may be added
-                hostDetectorSync,
-            ],
+            resourceDetectors: resolveDetectors(opts.resourceDetectors),
```

Comment:
Not for this PR. I wonder if eventually we may want to export something similar to `getInstrumentations()` that enables a user to user our SDK and get the default set of detectors **plus one or more custom detectors**.  Or perhaps this is something that is more generally solved with some concept of SDK plugins or extensions as briefly discussed on the OTel JS SIG.

Reply:
I'd say if the number of detectors grow enough we will have a case for that API to be exported. We can discuss it and crate an issue if we agree it's necessary